admin.py

```diff
@@ -1,25 +1,32 @@
 def add_drink(drinks, title, price):
-    drinks[title] = price
+    if title not in drinks:
+        drinks[title] = price
 
 
 def remove_drink(drinks, title):
-    drinks.pop(title)
+    if title in drinks:
+        drinks.pop(title)
 
 
 def change_price_drink(drinks, title, new_price):
-    drinks[title] = new_price
+    for title in drinks.keys():
```

Comment:
Access to the dictionary element is 1.
This is not a list. You don't need to go through all the dictionary keys to check if it exists.